package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "sigsa_reviews",
+  "version": "1.0.0",
+  "description": "Sigsa is a web app that simulates an OpenTable restaurant page experience. This reviews service is responsible for delivering content+data for the reviews module on the page.",
+  "author": "Priyanka Piali Pramanik",
+  "license": "ISC",
+  "engines": {
+    "node": ">=8.16.0"
+  },
+  "scripts": {
+    "pretest": "eslint --ignore-path .eslintignore --ext .js,.jsx .",
+    "react-dev": "webpack -d --watch",
+    "start-server": "nodemon server/index.js",
+    "start-client": "live-server public",
```

Comment:
since you finished your server side set up and you are using the nodemon, you don't need live-server.